packages/dashboard/android/app/build.gradle

```diff
@@ -52,7 +52,11 @@ react {
     // hermesFlags = ["-O", "-output-source-map"]
 }
 
-apply from: "../../node_modules/react-native-vector-icons/fonts.gradle"
+project.ext.vectoricons = [
```

Comment:
let's add a comment to every change we make to the default template. Something like:
```
// [super-app-showcase change]
```

This will help us upgrading the apps